src/instrumentation/logging.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 from typing import Dict, List, Any, Optional
 
-from src.config import QueryPlanConfig
+from src.config import RAGConfig
```

Comment:
Any reason for this change? QueryPlan seems more generic than RAGConfig if we add more operators etc..